fugue-aws/src/main/java/com/symphony/oss/fugue/aws/secret/AwsSecretManager.java

```diff
@@ -160,6 +160,41 @@ public IImmutableJsonDomNode getSecret(CredentialName name) throws SecretNotFoun
     }
 }
   
+  @Override
+  public String getSecretAsString(CredentialName name) throws SecretNotFoundException
```

Comment:
Have getSecret call this method and then parse the json to remove the duplicated code